kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/ImageServiceBase.java

```diff
@@ -144,8 +144,7 @@ public String getActiveProcessImage(String containerId, long procInstId, String
                     fullLogs.stream()
                             .filter(nodeInstanceDesc ->
                                             (((org.jbpm.kie.services.impl.model.NodeInstanceDesc) nodeInstanceDesc).getType() == NodeInstanceLog.TYPE_ASYNC_ENTER) &&
-                                                    fullLogs.stream().noneMatch(nodeInst -> nodeInstanceDesc.getNodeId().equals(nodeInst.getNodeId())
-                                                            && (((org.jbpm.kie.services.impl.model.NodeInstanceDesc) nodeInst).getType() == NodeInstanceLog.TYPE_EXIT)))
```

Comment:
@nmirasch @fjtirado could you please share a brief explanation about why we did check by type also?

Reply:
I see this comment ` // Async active nodes don't have any related completed node instance`

Reply:
@domhanak before this change it only checked that the async node has an type_exit log to include it as an active one. Now it checks that it's not in the finished ones (including Exited, Aborted, Skipped and Obsolete) to be included as an active node and painted properly.